src/main/java/core/basesyntax/operations/Action.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.service.TransactionParser;
+
+public interface Action {
+    boolean action(TransactionParser transaction);
```

Comment:
What means Transaction parser? Based on the class name I am expecting to see some business logic here. But in the method arguments here should be passed model or DTO